apps/passport-server/src/database/queries/telegram/deleteTelegramEvent.ts

```diff
@@ -49,15 +49,17 @@ export async function deleteTelegramChatTopic(
 
 export async function deleteTelegramForward(
   client: Pool,
-  senderChatTopicID: number,
-  receiverChatTopicID: number
+  receiverChatTopicID: number,
+  senderChatTopicID: number | null
 ): Promise<void> {
-  await sqlQuery(
-    client,
-    `\
-    delete from telegram_forwarding
-    where sender_chat_topic_id = $1 and receiver_chat_topic_id = $2
-    `,
-    [senderChatTopicID, receiverChatTopicID]
-  );
+  let queryString =
+    "DELETE FROM telegram_forwarding WHERE receiver_chat_topic_id = $1";
+  const queryParams = [receiverChatTopicID];
+
+  if (senderChatTopicID !== null) {
```

Comment:
enables the `stopreceive` command to stop receiving even if no topics are forwarding.